### PR TITLE
Automated Test - COMMERCE-6020 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**

### CODE REVIEW

**Comment by kristinonias** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase [hunk 1]
```diff
@@ -47,13 +47,186 @@ definition {
...
+
+		SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
+
+		Select(	
+			key_fieldLabel = "Price Type",
+			locator1 = "Select#GENERIC_SELECT_FIELD",
+			value1 = "Static");
+
+		Button.clickSave();
+
```
Comment (Pietro69) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase:229]: Is missing the Alert.viewSuccessMessage() (Step 17 into the ticket)

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase [hunk 2]
```diff
@@ -47,13 +47,186 @@ definition {
...
+		Click(
+			key_value = "Black",
+			locator1 = "CommerceEntry#BODY_VERTICAL_ELLIPSIS");
+
+		CommerceEntry.clickNoError(menuItem = "Delete");
+
+		Alert.viewSuccessMessage();
```
Comment (Pietro69) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase:236]: The test does not include these steps

**Review by Pietro69 — CHANGES_REQUESTED** [review]

**Review by Pietro69 — APPROVED** [review]

**Comment by Pietro69** [comment]
ci:test:sf

**Comment by Pietro69** [comment]
ci:test:relevant

**Comment by Pietro69** [comment]
ci:forward

**Pull request closed without merging**